### PR TITLE
[docs] Add publication list

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
@DevExpress/testcafe-docs

### CODE REVIEW

**Comment by VasilyStrelyaev** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
Errors will disappear once we merge https://github.com/DevExpress/testcafe-gh-page-assets/pull/37

File: docs/publications/publications.json [hunk 1]
```diff
@@ -0,0 +1,200 @@
+[
+    {
```
Comment (AlexanderMoskovkin) [inline docs/publications/publications.json]: Add please here the latest publication from dzone.com

**Comment by VasilyStrelyaev** [comment]
@testcafe-build-bot \retest

**Pull request merged**